sys/include/net/gnrc/sixlowpan/nd/router.h

```diff
@@ -199,7 +199,7 @@ void gnrc_sixlowpan_nd_router_abr_rem_prf(gnrc_sixlowpan_nd_router_abr_t *abr,
  * @brief   Adds a context for this border router to manage.
  *
  * @param[in] abr       The local border router
```

Comment:
Didn't touch this line and fixing all these inconsistencies in terms of periods at the end is another huge chunk of work,